### PR TITLE
Update front page

### PR DESCRIPTION
**Pull request by mor10**
## Description

Addresses issue #290

- Removes call to sidebar in `optional/front-page.php`
- Marginally improves the styling of the front page content when the template is used to display the blog index. 

## Todo
Get a designer to come up with a better look for the front page template.

## List of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] This pull request relates to a ticket.
- [x] This code is tested.
- [ ] This change has been added to CHANGELOG.md
- [x] I want my code added to WP Rig.

### CODE REVIEW

File: optional/front-page.php [hunk 1]
```diff
@@ -27,5 +27,5 @@
 		?>
 	</main><!-- #primary -->
 <?php
-get_sidebar();
+
```
Comment (bamadesigner) [inline optional/front-page.php]: Not a big deal but none of the other files have this extra line.

Comment (bamadesigner) [inline optional/front-page.php]: In Sidebars/component.php, its hard-coded to add "has-sidebar" selector to body class.

Comment (mor10) [reply optional/front-page.php]: No, the code in `Sidebars/component.php` does the opposite: it excludes the `has-sidebar` selector from this template. Unless I'm completely misreading the logic:

https://github.com/wprig/wprig/blob/v2.0/inc/Sidebars/Component.php#L90-L92

**Comment by mor10** [comment]
where are we on this?

**Comment by benoitchantre** [comment]
I’ll test that before Monday.

**Comment by benoitchantre** [comment]
@mor10 When comments are enabled, they are displayed next to the content, instead of below it.

File: optional/front-page.php (hunk 1 above)
Comment (bamadesigner) [reply optional/front-page.php]: You're right.

**Comment by mor10** [comment]
Ah. I see now. This template was originally designed to replace the front page as blog index, not the front page when set to display a single page. I need to figure out if I can hook into that logic. The grid only makes sense when the front page displays the blog index.

**Comment by mor10** [comment]
Updated PR with a conditional to test what is being displayed on the front page. With this update, the front page uses a grid only when the index is displayed. Please test and confirm so we can merge.

**Comment by benoitchantre** [comment]
Works for me.

Is there a reason to keep `front-page.css`? This file is not used on front-end anymore.

**Comment by bamadesigner** [comment]
Works for me. @benoitchantre, we need the front-page.css if y'all want these blog index styles that are showing up on the front-end.

**Comment by bamadesigner** [comment]
I don't see the comment issue.

**Comment by mor10** [comment]
@felixarntz do you want to resolve your complaint so we can merge?

**Comment by mor10** [comment]
Reviewing @felixarntz's comment, I think we're good. Merging.

**Pull request merged**